.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,3 +16,20 @@ Bullet points like
 really help with making this more readable.
 
 Fixes/Closes/Related-to #1 (enter issue number, except in rare cases where none exists).
+
+It is also helpful to add an emoji before the commit message to identify the kind of work done on a single commit. See the following suggestions belowf
+
+- :bug: - `:bug:` bug fix
+- :sparkles: - `:sparkles:` new feature
+- :exclamation: - `:exclamation:` tests
```

Comment:
+1 to :bike::house: :)

❗️ looks like something that really needs attention (like a high priority issue); is there something more neutral we can use?

Reply:
I think :white_check_mark: (`:white_check_mark:`) might work here.

Reply:
done.